bugbot/rules/assignee_no_login.py

```diff
@@ -88,6 +90,10 @@ def handle_inactive_assignees(self, bugs):
             self.add_action(bug)
             res[bugid] = bug
 
+            self.add(bug["assigned_to"], bug)
```

Comment:
We should add the triage owner as well. You could pass the emails as a list.

In most cases, the assignee will not receive an email because the nag emails will be sent only to Mozilla employees.

Reply:
Done in 138d65e09a93dacd5f57145673993f039a5b2688!